### PR TITLE
Added hasImage for both user and org. Also added adminDeleteEnabled to orgs

### PR DESCRIPTION
**Pull request by royanger**

### CODE REVIEW

File: docs/references/javascript/organization/organization.mdx [hunk 1]
```diff
@@ -16,8 +16,10 @@ This is a class that represents an organization. It's returned from the [`create
 | `slug` | `string \| null` | The organization slug. If supplied, it must be unique for the instance. |
 | `logoUrl` | `string \| null` | The full URL for the organization logo. (deprecated in favor of `imageUrl`) |
 | `imageUrl` | `string` | Holds the organization logo or default logo. |
+| `hasImage` | `boolean` | A getter boolean to check if the organization has a uploaded image. `false` if Clerk is displaying an avatar for the organization. |
```
Comment (alexisintech) [inline docs/references/javascript/organization/organization.mdx:19]: I think this prop comes up other times in the docs, we should update those instances to use this description as well.

```suggestion
| `hasImage` | `boolean` | A getter boolean to check if the organization has a uploaded image. Returns `false` if Clerk is displaying an avatar for the organization. |
```

File: docs/references/javascript/user/user.mdx [hunk 2]
```diff
@@ -23,6 +23,7 @@ The ClerkJS SDK provides some helper [methods](#methods) on the `User` object to
 | `fullName` | `string \| null` | The user's full name. |
 | `username` | `string \| null` | The user's username. |
 | `imageUrl` | `string` | Holds the users profile image or avatar. |
+| `hasImage` | `boolean` | A getter boolean to check if the user has uploaded an image or one was copied from OAuth. `false` if Clerk is displaying an avatar for the user. |
```
Comment (alexisintech) [inline docs/references/javascript/user/user.mdx:26]:
```suggestion
| `hasImage` | `boolean` | A getter boolean to check if the user has uploaded an image or one was copied from OAuth. Returns `false` if Clerk is displaying an avatar for the user. |
```

**Review by alexisintech — CHANGES_REQUESTED** [review]

File: docs/references/javascript/organization/organization.mdx [hunk 3]
```diff
@@ -16,8 +16,10 @@ This is a class that represents an organization. It's returned from the [`create
 | `slug` | `string \| null` | The organization slug. If supplied, it must be unique for the instance. |
 | `logoUrl` | `string \| null` | The full URL for the organization logo. (deprecated in favor of `imageUrl`) |
 | `imageUrl` | `string` | Holds the organization logo or default logo. |
+| `hasImage` | `boolean` | A getter boolean to check if the organization has a uploaded image. Returns `false` if Clerk is displaying an avatar for the organization. |
```
Comment (alexisintech) [inline docs/references/javascript/organization/organization.mdx:19]:
```suggestion
| `hasImage` | `boolean` | A getter boolean to check if the user has uploaded an image or one was copied from OAuth. Returns `false` if Clerk is displaying an avatar for the user. |
```

**Review by alexisintech — APPROVED** [review]
One minor thing for inconsistency. And then all good 😸💖

File: docs/references/javascript/organization/organization.mdx (hunk 3 above)
Comment (royanger) [reply docs/references/javascript/organization/organization.mdx:19]: This is not the user -- this is the organization. The 'an' is a correct edit, but this hasImage refers to the organization's image and not the user.

Comment (alexisintech) [reply docs/references/javascript/organization/organization.mdx]: oh good point, I was sifting through too quickly - thank you for catching that!

**Pull request merged**